tests/canonical-swap.ts

```diff
@@ -275,10 +276,111 @@ describe("canonical-swap", () => {
         wrappedData.publicKey
       );
 
-      expect(postTxWrappedData.paused).to.be.true;
+      expect(postTxWrappedData.swapCanonicalForWrappedEnabled).to.be.false;
     });
 
-    it("fails swap when paused", async () => {
+    it("fails swap when disabled", async () => {
+      const sourceTokenAccount = await createAccount(
+        provider.connection,
+        wallet.payer,
+        canonicalMint,
+        wallet.publicKey,
+        Keypair.generate()
+      );
+
+      const destinationTokenAccount = await createAccount(
+        provider.connection,
+        wallet.payer,
+        wrappedMint,
+        wallet.publicKey,
+        Keypair.generate()
+      );
+
+      const destinationAmount = BigInt(10);
+      const sourceAmount = destinationAmount * BigInt(10);
+
+      await transfer(
+        provider.connection,
+        wallet.payer,
+        tokenDistributorTokenAccount,
+        sourceTokenAccount,
+        wallet.publicKey,
+        sourceAmount
+      );
+
+      const failedSwap = canSwap.methods
+        .swapCanonicalForWrapped(
+          new BN(destinationAmount),
+          wrappedTokenAccountAuthorityBump
+        )
+        .accounts({
+          user: wallet.publicKey,
+          sourceCanonicalTokenAccount: sourceTokenAccount,
+          canonicalMint: canonicalMint,
+          destinationWrappedTokenAccount: destinationTokenAccount,
+          wrappedTokenAccount,
+          pdaWrappedTokenAuthority: wrappedTokenAccountAuthority,
+          canonicalData: canonicalData.publicKey,
+          wrappedData: wrappedData.publicKey,
+          tokenProgram: TOKEN_PROGRAM_ID,
+        })
+        .signers([wallet.payer])
+        .rpc();
+
+      await expect(failedSwap).to.eventually.be.rejectedWith(
+        "AnchorError caused by account: wrapped_data. Error Code: ConstraintRaw. Error Number: 2003. Error Message: A raw constraint was violated."
+      );
```

Comment:
Maybe worth adding an assert that swapWrappedForCanonical still succeeds here (and vice versa in `#swapWrappedForCanonicalEnabled` below)